### ISSUE TITLE
And condition not working 

### ISSUE THREAD

**Issue by guide5232** [YetiForceCompany/YetiForceCRM#9487]
# 🐞 bug report:
I have written workflow like this 

![workflow system settings1](https://user-images.githubusercontent.com/47585969/52763646-bf920900-3070-11e9-8153-99c5e8356975.png)

![workflow system settings3](https://user-images.githubusercontent.com/47585969/52763997-43002a00-3072-11e9-8118-ef037e9eb8bf.png)

In this example, we will configure a workflow to escalate a case, if it’s not responded for more than four hours. You can have the workflow either escalate the case to the support manager, or trigger an email alert, or do both. This workflow is executed once in every hour,but this notification is going to manager every one hour. actually i want to send a record details for manager after 4 hour from record created time . please give me suggestions 

### 🌍  Your Environment 

| Environment		|	Version / Name	|
| ----------------	|	--------------	|
| YetiForce			| 
| Web server (name and version)|		|
| PHP  |								|
| Browser (name and version)  | 		|
| Operating System (name and version)  | 		|
| Database (name and version) | 		|

**Comment by vovpff** [comment]
@guide5232 Try `Only once...` option

**Comment by guide5232** [comment]
I Have checked only once one option but it is not firing without changing record.i want to fire  automatically any workflow.

**Comment by vovpff** [comment]
@guide5232 I see as solution 2 workflows. First one every four updates some selfmade hidden system field. Second one as issue subject.

**Comment by guide5232** [comment]
Sorry i didn't get  your point please give some more explanation

**Comment by vovpff** [comment]
Add value field and hourly add to initial value +1 by workflow.. Hide this field from all users, After some time your workflow from first post with `only once` condition will trigger all listed condition and send notification.

**Comment by guide5232** [comment]
![workflow system settings4](https://user-images.githubusercontent.com/47585969/52774980-b1ed7b00-3092-11e9-8453-a1a975f24409.png)

 I have  created one value field with time data type  in module .later that value field i have assign +1 in workflow condition  please tell me is it correct or not

**Comment by vovpff** [comment]
@guide5232 not so. You need `update field` task with expression for value field like `value + 1`.

**Comment by vovpff** [comment]
@guide5232 I has some brainstorming and found next theoretical solution. Forget about all writen before. You need to add some field like a check box with some name. After that you hourly check your tickets with conditions as on picture
![workflow system settings3](https://user-images.githubusercontent.com/47585969/52763997-43002a00-3072-11e9-8118-ef037e9eb8bf.png)
plus you need to check this checkbox state (false)
For example if all conditions met (time, status, priority, checkbox is false) you need two tasks for workflow. 1st send notification, 2nd change checkbox state to `true`. Next scheduled check will not trigger notification sending because your checkbox is true.

**Comment by guide5232** [comment]
Ok thanks
![workflow system settings2](https://user-images.githubusercontent.com/47585969/52778453-28da4200-309a-11e9-954a-bb7f2dc55cdb.png)

This is condition not wait for complete 4 hours , it is firing the before 4 hours

**Comment by vovpff** [comment]
@guide5232 your last comment should be new issue

**Comment by guide5232** [comment]
Please tell me how can i achieve above requirement in this Yetiforce CRM

**Comment by vovpff** [comment]
At first you need to find hosting or establish LAMP server

**Comment by guide5232** [comment]
![server configuration system settings](https://user-images.githubusercontent.com/47585969/52834003-b28a1e00-3133-11e9-8e05-436bc37d2970.png)

This my sever time zone where can i change that server time zone from Europe to Australia time zone

**Comment by vovpff** [comment]
You can change in your php settings. This is parameter of your webserver

**Comment by guide5232** [comment]
If i change time zone Then "more than hours before" and "more than hours later" working properly or not

**Comment by mariuszkrzaczkowski** [comment]
config/Main.php
```php
	/** Set the default timezone as per your preference */
	public static $default_timezone = 'Europe/Warsaw';
```

**Comment by guide5232** [comment]
Still time dependency workflow not working. please help me how to give condition for  created time workflow

**Comment by areksolek** [comment]
If you want to send the details of the record then you must set: "Creating a record" with the condition:
Field "Creation time" - "more than hours later" - 4.
Then by the time of creation to give up 4 hours.
A similar problem created by you has been solved here # 9592.
